docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,313 @@
+data, ``LTS_AUTHORS.cff``. This file will be used to create the associated ``codemeta.json`` file.
+
+5. Possible implementation
+==========================
+
+5.1 DCO implementation
+----------------------
+
+Today, the tool offered by GitHub to sign the DCO is to add the extra parameter ``-s`` to each git commit. Users have to
+be aware of that and get used of it. For the reviewers, a github CI can be used to check quickly if a contributor has
+signed the DCO. This method will be used for the moment. If there is any other method that avoids this extra parameter,
+one will naturally consider it.
+
+In order to respect these rules, some automation is required to create the list of contributors. One could use
+the Python library `Tributors <https://con.github.io/tributors/>`_. This library uses the GitHub API to determine the
+list of contributors, which is written in the format associated with codemeta.json, citation.cff, zenodo.json. This kind
+of library allows then two requirements: retrieve automatically the list of contributors from GitHub and write the
+authors list in all the needed formats.
+
+5.2 Collection of the personal information of authors
+-----------------------------------------------------
+
+The data collected from the git logs are not sufficient for any publication. Most of the contributors must give their
+affiliation as stipulated by their working contract. Also, the ORCID number is a relatively new type of information,
+recommended to be used in the context of the Open Science movement, but it is not mandatory. However, one needs to
+collect this information to build the authors lists, in a safe and fair way.
+
+In this purpose, one could use the public user profile of the GitHub platform. They are public data, but one should
+aware that they are not complete and/or up-to-date. As mention earlier, it is recommended to fill correctly this GitHub
+profile to help the dev team to pre-filled the metadata files.
+
+One should note that the GitHub profile does not contain a field associated with the ORCID
+identifier. However, one could also retrieve the ORCID identifier of a contributor with the ORCID API
+(e.g. `How-To-1 <https://stackoverflow.com/questions/71052912/extracting-credentials-from-orcid-seach-with-orcid-id-using-python>`_
+or `How-To-2 <https://info.orcid.org/documentation/api-tutorials/api-tutorial-searching-the-orcid-registry/#easy-faq-2532>`_).
+
+In any case, a specific Python script located in the main Gammapy repository had to be written such
+that the authors' order follows these rules. In this case, integration within a GitHub action could be possible.
+
+In this context, one could technically use an other cff file as maintenance file to store the personal data of users
+that have signed the DCO.
+
+A second script should be settled in order to create the list of contributors in HTML format that should be
```

Comment:
How frequently is the team description planned to be updated? What is on the web page might be different from the releases/papers?

Reply:
Note that this list should not contain any personal data other than the name I suppose